common/pb/moderator_commands.proto

```diff
@@ -24,4 +27,30 @@ message Command_GetBanHistory {
         optional Command_GetBanHistory ext = 1001;
     }
     optional string user_name = 1;
+}
+
+message Command_WarnUser {
+    extend ModeratorCommand {
+        optional Command_WarnUser ext = 1002;
+    }
+
+    optional string user_name = 1;
+    optional string reason = 2;
+    optional string clientid = 3;
+}
+
+message Command_GetWarnHistory {
+    extend ModeratorCommand {
+        optional Command_GetWarnHistory ext = 1003;
+    }
+    optional string user_name = 1;
+}
+
+message Command_GetWarnList {
```

Comment:
I'm having a hard time understanding what this is for vs GetWarnHistory. Can you explain the difference? Comments on these would help